CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [7.0.2] - 2020-07-22
 ### Removed
 - Removed PV node limits for AWS as the feature gate is no longer supported in 1.17+.
```

Comment:
Isn't that a breaking change then?

Reply:
I don't think so

Reply:
This is funny. I get where Pawel comes from. In this case we actually remove support for something in order to prepare for a breaking change that we just have to adhere to in order to keep our library functional and maintainable.